pkg/system/reconciler.go

```diff
@@ -284,6 +287,7 @@ func NewReconciler(
 
 	r.KedaTriggerAuthentication.Namespace = r.Request.Namespace
 	r.KedaScaled.Namespace = r.Request.Namespace
+	r.AdapterHPA.Namespace = r.Request.Namespace
```

Comment:
move namespace set to the namespace area for all of this resources - line 220
also add names to the names set area line 263, following our format: r.Request.Name + <type>
```
	r.AdapterHPA.Name = r.Request.Name + "-adapter-hpa"
	r.KedaScaled.Name = r.Request.Name + "-keda-scaled"
```

Reply:
@jackyalbo code updated,